metals/src/main/resources/db/migration/V3__Jar_symbols.sql

```diff
@@ -1,6 +1,6 @@
 -- Indexed jars, the MD5 digest of path, modified time and size as key
 create table indexed_jar(
-  id int auto_increment,
+  id int auto_increment unique,
   md5 varchar primary key
 );
```

Comment:
If the database has to be created from scratch can we also merge all migrations into one? What do you think?

Reply:
I think it's better to leave it as is, since we will get the proper git history (why was this added and when). I don't see any clear benefit to migrating them to one file.